lms/envs/common.py

```diff
@@ -1257,7 +1257,7 @@
     'js/vendor/url.min.js',
     'common/js/vendor/underscore.js',
     'common/js/vendor/underscore.string.js',
-    'common/js/vendor/picturefill.min.js',
```

Comment:
Yep, that's the pattern we'd like to move towards as we pull things out of `/vendor` and begin installing them via NPM.